app/src/main/java/com/poupa/vinylmusicplayer/service/MusicService.java

```diff
@@ -619,6 +620,35 @@ public void onResourceReady(@NonNull Bitmap resource, Transition<? super Bitmap>
         }
     }
 
+    public void updateMediaSessionLyric(String line) {
+        final Song song = getCurrentSong();
+
+        if (song.id == -1) {
```

Comment:
Better use `song.id == s=Song.EMPTY.id`, ie dont assume implicitly the use of -1 constant.